software/snpeff/build.sh

```diff
@@ -0,0 +1,34 @@
+export GENOME=GRCh37
```

Comment:
Can we have a `for` loop here that goes through these entries? Not sure if you can have a list of lists type structure as you would in Python.